### ISSUE TITLE
[Bug]: Pip installation fails for BT-MQTT-Gateway

### ISSUE THREAD

**Issue by TJuTZu** [FaserF/hassio-addons#260]
### The problem

Update of bt-mqtt-gateway from 1.3.8 -> 1.3.9 fail on Home Assistant Green.
and Installation of bt-mqtt-gateway 1.3.9. fail after 1.3.8 was removed.

### What version of Home Assistant Core has the issue?

2024.1.6

### What version of the addon has the issue?

1.3.9

### What type of installation are you running?

Home Assistant OS

### Add-On causing the issue

BT-MQTT-Gateway

### Log information

```txt
Failed to install add-on
The command '/bin/ash -o pipefail -c cd /application && apk add --no-cache py3-pip && pip install --no-cache-dir -r requirements.txt && pip install --no-cache-dir `./gateway.py -r all`' returned a non-zero code: 1
```

### Additional information

|||
|---|---|
| Core | 2024.1.6 |
| Supervisor | 2023.12.1 |
| Operating System | 11.4 |
| Frontend | 20240104.0 |

**Comment by Micha70** [comment]
Hello, I have the same issue. Update from 1.3.8 to 1.3.9 fails. Using also 
Core 2024.1.6
Supervisor 2023.12.1

**Comment by FaserF** [comment]
Fixed now with 1.4.0

**Comment by Micha70** [comment]
@TJuTZu, does the add on is usable on your side? I can install now, but is not running, see https://github.com/FaserF/hassio-addons/issues/263
